File: codeforces/ComputerGame.py
def computerGame(grid):
    output = False

    for i in range(len(grid[0])):
        output |= grid[0][i] == '1' and grid[1][i] == '1'
    if not output:
        return "YES"
    return "NO"


def main():
    t = int(input())
    for _ in range(t):
        n = int(input())
        arr = []
        for _ in range(2):
            arr.append(list(input()))
        print(computerGame(arr))


if __name__ == "__main__":
    main()

File: codeforces/ComputerGame.c
#include <stdio.h>
#include <stdbool.h>

void computerGame();
int main(void)
{
    int t;
    scanf("%d", &t);
    for (int i = 0; i < t; i++)
    {
        computerGame();
    }
    return 0;
}
void computerGame()
{
    int n;
    bool output = false;
    scanf("%d", &n);
    char col1[n];
    char col2[n];
    scanf("%s", col1);
    scanf("%s", col2);
    for (int i = 0; i < n; i++)
        output |= (col1[i] == '1' && col2[i] == '1');
    if (output)
        printf("NO\n");
    else
    {
        printf("YES\n");
    }
}